<launch>

  <arg name="interface" default="can0" />
  <arg name="receiver_interval_sec" default="0.01" />
  <arg name="sender_timeout_sec" default="0.01" />
  <arg name="enable_can_fd" default="false" />
  <arg name="from_can_bus_topic" default="from_can_bus" />
  <arg name="to_can_bus_topic" default="to_can_bus" />

  <include file="$(find-pkg-share ros2_socketcan)/launch/socket_can_receiver.launch.py">
    <arg name="interface" value="$(var interface)" />
    <arg name="interval_sec" value="$(var receiver_interval_sec)" />
    <arg name="enable_can_fd" value="$(var enable_can_fd)" />
    <arg name="from_can_bus_topic" value="$(var from_can_bus_topic)" />
  </include>

  <include file="$(find-pkg-share ros2_socketcan)/launch/socket_can_sender.launch.py">
    <arg name="interface" value="$(var interface)" />
    <arg name="timeout_sec" value="$(var sender_timeout_sec)" />
    <arg name="enable_can_fd" value="$(var enable_can_fd)" />
    <arg name="to_can_bus_topic" value="$(var to_can_bus_topic)" />
  </include>

</launch>
